src/util.ts

```diff
@@ -27,6 +27,27 @@ export function intListToStringList(jsonStruct: string): string {
     return jsonStruct.replace(/(\-?[0-9]+)/g, '"$1"');
 }
 
+/**
+ * Replaces a string in a JSON string with the same string as a
+ * number, i.e. removing quotes (") prior to and after the string. This
+ * is needed as the default JSON stringify cannot serialize BigInts as numbers.
+ * So one can turn them into strings, stringify the structure, and then use this function
+ * to make those fields into JSON numbers.
+ * @param jsonStruct the JSON structure as a string
+ * @param keys the keys where the strings has to be unquoted
+ * @returns the same JSON string where the strings at the supplied keys are unquoted
+ */
+export function stringToInt(jsonStruct: string, keys: string[]): string {
```

Comment:
Add unit tests for this method. On top of that we should have a unit test that uses `intToString` and `stringToInt` too.